archunit/src/main/java/com/tngtech/archunit/core/domain/JavaMethod.java

```diff
@@ -70,11 +70,28 @@
         return annotationDefaultValue;
     }
 
+    @Override
+    @PublicAPI(usage = ACCESS)
+    public boolean isConstructor() {
+        return false;
```

Comment:
Same here, the default is `false` anyway. As long as we go with a default (which you could question :man_shrugging:) we should use it then I guess...